src/tequila/quantumchemistry/qc_base.py

```diff
@@ -1638,7 +1638,8 @@ def rdm2(self):
             return None
 
     def compute_rdms(self, U: QCircuit = None, variables: Variables = None, spin_free: bool = True,
-                     get_rdm1: bool = True, get_rdm2: bool = True, ordering="dirac", use_hcb: bool = False):
+                     get_rdm1: bool = True, get_rdm2: bool = True, ordering="dirac", use_hcb: bool = False,
+                     transformation: QCircuit = False):
```

Comment:
transformation has to be of type `QubitHamiltonian` otherwise it won't work
(the H*U used later on does not work if U is a circuit - in principle it could, but it's not implemented)

you use it correctly, as you are using tq.paulis.X instead of tq.gates.X